tests/wpt/metadata/fetch/api/response/response-stream-disturbed-5.html.ini

```diff
@@ -1,5 +1,3 @@
 [response-stream-disturbed-5.html]
   type: testharness
-  [Getting a body reader after consuming as arrayBuffer]
-    expected: FAIL
```

Comment:
Shouldn't this file be deleted altogether?

Reply:
It will be automatically in the future.